src/hotspot/os/windows/perfMemory_windows.cpp

```diff
@@ -54,7 +54,7 @@ typedef BOOL (WINAPI *SetSecurityDescriptorControlFnPtr)(
 static char* create_standard_memory(size_t size) {
 
   // allocate an aligned chuck of memory
-  char* mapAddress = os::reserve_memory(size);
+  char* mapAddress = os::reserve_memory(size, mtNone);
```

Comment:
To match with the other platforms:
```suggestion
  char* mapAddress = os::reserve_memory(size, mtInternal);
```